assets/scss/application.scss

```diff
@@ -7,6 +7,8 @@ $govuk-page-width: $moj-page-width;
 
 @import 'govuk-frontend/dist/govuk';
 @import '@ministryofjustice/frontend/moj/all';
+@import '../../node_modules/accessible-autocomplete/dist/accessible-autocomplete.min';
```

Comment:
i think the import path should be `'accessible-autocomplete/dist/accessible-autocomplete.min'` because the node modules folder is already included in the sass load paths and because a relative import is more brittle – it depends more strongly on where this file lives.